test/LibraryManager.IntegrationTest/VisualStudioLibmanHostTests.cs

```diff
@@ -38,13 +41,51 @@ protected override void DoHostTestInitialize()
 
             Solution.Open(_solutionPath);
             Solution.WaitForFullyLoaded(); // This will get modified after bug 627108 get fixed
+
+            string pathToLibmanFile = Path.Combine(SolutionRootPath, _projectName, _libman);
+            _initialLibmanFileContent = File.ReadAllText(pathToLibmanFile);
         }
 
         protected override void DoHostTestCleanup()
```

Comment:
>DoHostTestCleanup() [](start = 32, length = 19)

Do we want to have this clean up after each test running or just have TestCleanup() in individual test class so that it will not getting executed for no library changed tests like LibmanCompletionTests.

Reply:
Cleaning up after every test ensures that no test can leave it in a dirty state.  Any test which opens/types in this file has a chance of leaving it dirty if something fails or throws during execution.

Reply:
I don't want to include these in every test. We could override in LibmanCompletionTests. But clean won't do much here since the initial contents has no libraries. So its not expensive. Let me know if you disagree



---
In reply to: [208043077](https://github.com/aspnet/LibraryManager/pull/341#discussion_r208043077) [](ancestors = 208043077)

Reply:
If this wasn't in response to Yang's comment, let me know

---
In reply to: [208044697](https://github.com/aspnet/LibraryManager/pull/341#discussion_r208044697) [](ancestors = 208044697)